web3.js/src/connection.ts

```diff
@@ -4278,6 +4286,26 @@ export class Connection {
     }
   }
 
+  /**
+   * get the stake minimum delegation
+   */
+  async getStakeMinimumDelegation(
+    commitmentOrConfig?: Commitment | GetStakeMinimumDelegationConfig,
```

Comment:
so i found that many of such calls were also providing an option to give commitment directly or a config (with commitment and other properties) , so i implemented it like that, if this is something needed for this also.